docs/components/dartpad/top-app-bar/theme/main.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
+      home: MyHomePage(),
+      theme: _buildShrineTheme(),
+    );
+  }
+}
+
+class MyHomePage extends StatelessWidget {
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        leading: Icon(Icons.menu),
+        title: Text('Page title'),
+        actions: [
+          Icon(Icons.favorite),
+          Padding(
+            padding: EdgeInsets.symmetric(horizontal: 16),
+            child: Icon(Icons.search),
+          ),
+          Icon(Icons.more_vert),
+        ],
+      ),
+      body: Container(),
+    );
+  }
+}
+
+ThemeData _buildShrineTheme() {
```

Comment:
Instead of having Shrine theme defined twice, it make more sense to have a common file to source all Shrine theming from

Reply:
That would be nice but Dartpad doesn't give us that freedom